### PR TITLE
US6912 background color helper classes

### PR DESCRIPTION
**Pull request by seancdavis**
Add helper classes for background colors to DDK

Examples of this include the live stream page, for setting background color on modal or sticky header

---

Corresponds with crdschurch/crds-styleguide#41

### CODE REVIEW

File: assets/stylesheets/bootstrap.scss [hunk 1]
```diff
@@ -5,3 +5,4 @@
 @import 'base/modules';
 @import 'components/modules';
 @import 'themes/modules';
+@import 'utilities/modules';
```
Comment (tcmacdonald) [inline assets/stylesheets/bootstrap.scss]: Does it make more sense to do this and refactor @zarahzachz's newly added helper classes into `utilities/modules` or move these classes into her file? 

https://github.com/crdschurch/crds-styles/blob/development/assets/stylesheets/base/_utilities.scss

Comment (seancdavis) [inline assets/stylesheets/bootstrap.scss]: Good question. What do you think, @zarahzachz?

I created the new directory following [the article you sent me yesterday](https://scotch.io/tutorials/aesthetic-sass-1-architecture-and-style-organization). I'd be happy to take on refactoring the current utilities into their own modules, unless you think it makes more sense to keep them in a single file?

Comment (zarahzachz) [inline assets/stylesheets/bootstrap.scss]: I actually forgot about the `utils/` directory. I'm fine moving my work into the directory @seancdavis created, but it should probably stay 1 file (at least for now).

Comment (seancdavis) [inline assets/stylesheets/bootstrap.scss]: OK. If we want to keep just one file, I'll just leave it where it is now and then when can move to `utils/` when it's appropriate to break into individual files.

**Pull request merged**